src/modules/addTask.js

```diff
@@ -0,0 +1,28 @@
+import itemsArray from './itemsArray.js';
+import renderToDoItems from './renderTasks.js';
+
+const addElBtn = document.querySelector('.to-do-addTask');
+const input = document.querySelector('input');
+const todoinputEl = document.querySelector('.to-do-input');
+
+const addTask = () => {
+  addElBtn.addEventListener('click', () => {
+    const newTask = todoinputEl.value;
+    if (newTask) {
+      const newTodo = {
+        description: newTask,
+        completed: false,
+      };
+      itemsArray.push(newTodo);
+      localStorage.setItem('itemsArray', JSON.stringify(itemsArray));
+      renderToDoItems();
+      todoinputEl.value = '';
+    } else {
+      input.classList.add('inputError');
+      setTimeout(() => {
+        input.classList.remove('inputError');
+      }, 1000);
+    }
+  });
+};
```

Comment:
- [ ] Kindly store the task `index` when you add a new task and set the index value with the new array length to achieve this [requirement](https://github.com/microverseinc/curriculum-javascript/blob/main/todo-list/m3_add_remove.md#:~:text=By%20default%20new%20tasks%20should%20have%20the%20property%20completed%20set%20to%20false%20and%20the%20property%20index%20set%20to%20the%20value%20of%20the%20new%20array%20length%20(i.e.%20if%20you%27re%20adding%20a%205th%20task%20to%20the%20list%2C%20the%20index%20of%20that%20task%20should%20equal%20to%205).).

Reply:
- Just like the previous reviewers have suggested, and according to the project requirements, you are required to have an  `index` key when a task is added to your to-do list. I noticed this hasn't been implemented yet. Please its very important you fix this 🙏